### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by chingor13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 This library provides an [`ExporterInterface`][exporter-interface] for exporting
 Trace data to a Zipkin instance.
 
-[![CircleCI](https://circleci.com/gh/census-instrumentation/opencensus-php-exporter-zipkin.svg?style=svg)](https://circleci.com/gh/census-instrumentation/opencensus-php-exporter-zipkin)
-[![Packagist](https://img.shields.io/packagist/v/opencensus/opencensus-exporter-zipkin.svg)](https://packagist.org/packages/opencensus/opencensus)
+[![CircleCI](https://circleci.com/gh/census-instrumentation/opencensus-php-exporter-zipkin.svg?style=svg)][ci-build]
```
Comment (tmatsuo) [inline README.md]: path with the new org?

File: README.md [hunk 2]
```diff
@@ -73,3 +99,5 @@ This is not an official Google product.
 [census-org]: https://github.com/census-instrumentation
 [composer]: https://getcomposer.org/
 [semver]: http://semver.org/
+[ci-build]: https://circleci.com/gh/census-instrumentation/opencensus-php-exporter-zipkin
```
Comment (tmatsuo) [inline README.md]: ditto

File: README.md (hunk 1 above)
Comment (chingor13) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (chingor13) [reply README.md]: Fixed.

**Review by tmatsuo — APPROVED** [review]

**Pull request merged**